python/tank/authentication/login_dialog.py

```diff
@@ -371,7 +371,7 @@ def _toggle_web(self):
         # We only update the GUI if there was a change between to mode we
         # are showing and what was detected on the potential target site.
         use_web = (
-            self._query_task.sso_enabled or self._query_task.autodesk_identity_enabled
+            self._query_task.sso_enabled
```

Comment:
Only ShotGrid-SSO is left as the one authentication mode which absolutely requires a web environment to authenticate.